src/components/NcRichText/autolink.js

```diff
@@ -115,16 +115,19 @@ export const parseUrl = (text) => {
  * @param {import('vue-router').default} router - VueRouter instance of the router link
  * @param {string} url - absolute URL to parse
  * @return {string|null} a path that can be useed in the router link or null if this URL doesn't match this router config
```

Comment:
```suggestion
 * @return {string|null} a path that can be used in the router link or null if this URL doesn't match this router config
```

Reply:
Installed Spell Checker extension to VSCode as well 🙈